### PR TITLE
VsIG-IElement-1

### PR DESCRIPTION
**Pull request by VsIG-official**

### CODE REVIEW

**Comment by VsIG-official** [comment]
- About Wall:
1. I've created enum **Orientation** with _Horizontal_, _Vertical_ and I thought, that _Cross_ would be good too
2. I think _Player_ isn't necessary in this class, 'cause, if We want to track number of walls, then We can place some counter, like _WallsLeft_ or _WallsNumber_ (default value = 10) to the **Player** and decrement it from _CurrentPlayer_ in Quoridor or other class, which will handle this logic

- About Cell
1. We have bool _HasPlayer_, but maybe it would be better, if We'll replace _bool_ with _Player_ type and set it's value to the first, second or nullObject Player

3/2. (this statement is for both of these classes) Also method _Place_ has bool return type (just in case), but We can make it void later, if We want

**Comment by VsIG-official** [comment]
after Your review's I'll change model table appropriate to the chops

**Review by Anastasia-Tre — APPROVED** [review]

File: Quoridor.Model/Cell.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿namespace Quoridor.Model
+{
+	internal class Cell : IElement
+	{
```
Comment (Anastasia-Tre) [inline Quoridor.Model/Cell.cs:4]: Can we add a field for id, for better implementation of search path algorithm?
```suggestion
	{
	    public int Id { get; private set; }
```

**Comment by VsIG-official** [comment]
@Anastasia-Tre , Your's suggestion is applied, but maybe We need to add this into constructor too?

**Review by LovelyDrug — APPROVED** [review]

**Comment by Anastasia-Tre** [comment]
@VsIG-official , yeah, it is good idea to add this field to constructor

**Pull request merged**